certified-connectors/nps.today/apiDefinition.swagger.json

```diff
@@ -0,0 +1,788 @@
+            },
+            "SurveyOpenedTime": {
+              "type": "string",
+              "description": "SurveyOpenedTime"
+            },
+            "SurveySendTime": {
+              "type": "string",
+              "description": "SurveySendTime"
+            },
+            "SurveyScheduleTime": {
+              "type": "string",
+              "description": "SurveyScheduleTime"
+            },
+            "Respondent": {
+              "type": "object",
+              "properties": {
+                "Id": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Id"
+                },
+                "Address1": {
+                  "type": "string",
+                  "description": "Address1"
+                },
+                "Address2": {
+                  "type": "string",
+                  "description": "Address2"
+                },
+                "City": {
+                  "type": "string",
+                  "description": "City"
+                },
+                "Country": {
+                  "type": "string",
+                  "description": "Country"
+                },
+                "ZipCode": {
+                  "type": "string",
+                  "description": "ZipCode"
+                },
+                "EmailAddress": {
+                  "type": "string",
+                  "description": "EmailAddress"
+                },
+                "FirstName": {
+                  "type": "string",
+                  "description": "FirstName"
+                },
+                "LastName": {
+                  "type": "string",
+                  "description": "LastName"
+                },
+                "Active": {
+                  "type": "boolean",
+                  "description": "Active"
+                },
+                "Title": {
+                  "type": "string",
+                  "description": "Title"
+                },
+                "CompanyName": {
+                  "type": "string",
+                  "description": "CompanyName"
+                },
+                "EmployeeId": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "EmployeeId"
+                },
+                "EmployeeEmail": {
+                  "type": "string",
+                  "description": "EmployeeEmail"
+                },
+                "Department": {
+                  "type": "string",
+                  "description": "Department"
+                },
+                "PhoneNumber": {
+                  "type": "string",
+                  "description": "PhoneNumber"
+                },
+                "ExternalId": {
+                  "type": "string",
+                  "description": "ExternalId"
+                },
+                "EmployeeExternalId": {
+                  "type": "string",
+                  "description": "EmployeeExternalId"
```

Comment:
@HazemSaeid The description values of properties should not be the property ids. Can you please provide valid description values for all the propeties.

Documentation - [Description values of properties](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Resolved in latest commit